README.md

```diff
@@ -83,7 +83,6 @@ https://github.com/openenclave/openenclave (branch: feature/sgx-lkl-support).
 1. Clone the `feature.sgx-lkl` branch of the Open Enclave SDK:
 ```
 git clone -b feature/sgx-lkl-support git@github.com:openenclave/openenclave.git openenclave-sgxlkl.git
```

Comment:
While you're at it, can you remove `.git` from the path?

Reply:
Done